lib/rules/no-confusing-arrow.js

```diff
@@ -54,6 +55,7 @@ module.exports = {
     create(context) {
         const config = context.options[0] || {};
         const allowParens = config.allowParens || (config.allowParens === void 0);
+        const onlyOneSimpleParam = !!config.onlyOneSimpleParam;
```

Comment:
```suggestion
        const onlyOneSimpleParam = config.onlyOneSimpleParam;
```

No need for !! Because this is always a Boolean.